pages/index.mdx

```diff
@@ -53,7 +53,7 @@ As your service grows, GOV.UK PaaS lets you instantly:
 GOV.UK PaaS is designed to meet the needs of government teams working on public-facing services:
 
 -   UK hosted
--   It provides 99.99% uptime (based on the last six months' [performance](https://status.cloud.service.gov.uk/))
+-   It provides [99.99% uptime (based on the last six months' performance)](https://status.cloud.service.gov.uk/)
```

Comment:
Parenthesis inside `[]` is not correct?

Reply:
is it not https://github.com/alphagov/paas-product-pages/blob/290603f4002529508cfad60bb97bca09a06a9c44/pages/index.mdx ? 
i know I have bad eyes sight..

Reply:
no it is I who have bad eyesight